srcpkgs/libxml++/template

```diff
@@ -1,28 +1,24 @@
-# Template file for 'libxml++'.
+# Template file for 'libxml++'
 pkgname=libxml++
```

Comment:
Why don't you make this libxml++3.0 instead of renaming 2.6?

Reply:
3.0 is the latest version and 2.6 should be dropped when all packages support it.

Reply:
Which isn't going to happen (or at least not any time soon, most of those projects are dead (like libxml++)). We also have python and python3 instead of python2 and python. IMO we should avoid unnecessary renames.